code/game/machinery/doors/airlock.dm

```diff
@@ -310,17 +310,7 @@ GLOBAL_LIST_EMPTY(wedge_icon_cache)
 	var/last_event = 0
 
 /obj/machinery/door/airlock/Process()
-	// Deliberate no call to parent.
```

Comment:
Airlocks are one of the few things that aren't terribly buggy right now. If this breaks airlocks I will be very, very mad. This code is extensively tested by you right Tori?